vuepress/docs/next/docs/getting-started/ent-api.md

```diff
@@ -1,4 +1,72 @@
+---
+sidebarDepth: 1
+---
 
 
+# Overview
 
+An API claim informs the Entando Platform that a microfrontend (MFE) or microservice (MS) intends to use the API of another microservice. This request can be sent within a single bundle or across bundles. The abstraction of an API claim eliminates the need to define and manage API endpoints, both in local development and within a running instance.
 
+## Internal vs. External API Claim
+
+An MFE can initiate both internal and external API claims. An internal API claim specifies an MS API existing in the same bundle as the MFE, while an external API claim specifies the MS API of another bundle.
+
+The structure of internal and external API claims are as follows:
+
+**Internal API claim**
+``` json
+{
+    "name": "int-api",
+    "type": "internal",
+    "serviceName": "int-ms"
+}
+```
+**External API claim**
+``` json
+{
+    "name": "ext-api",
+    "type": "external",
+    "serviceName": "ext-ms",
+    "bundle": "registry.hub.docker.com/my-org/my-entando-bundle"
+}
+```
+
+## API Claim Commands
+
+Common operations associated with API claims are detailed below.
```

Comment:
Might be good to show an example of the full command (ent bundle api) somewhere?